CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 * The `apos.util.attachmentUrl` method now works correctly. To facilitate that, `apos.uploadsUrl` is now populated browser-side at all times as the frontend logic originally expected. For backwards compatibility `apos.attachment.uploadsUrl` is still populated when logged in.
 * Widget players are now prevented from being played twice by the implementing vue component.
+* The`fieldsPresent` method that is sued with the `presentFieldsOnly` option in doc-type was broken, looking for properties in strings and wasn't returning anything.
```

Comment:
"sued" should be "used"

Reply:
This is currently in the 3.8.0 section. It should be in the Unreleased section.

Reply:
Oups yes thanks.